e2e/scripts/st_plotly_chart.py

```diff
@@ -42,7 +42,9 @@
 
 # Bubble Chart
 # Tests Discrete coloring with streamlit theme
-st.plotly_chart(fig_bubble, theme="streamlit")
+# Tests use container width when updating layout
+fig_bubble.update_layout(height=300, width=300)
+st.plotly_chart(fig_bubble, use_container_width=True, theme="streamlit")
```

Comment:
Checks for use_container_width=True when height and width are changed.

Reply:
To reword: 
This checks that we use the container width even when the height and width are changed and the streamlit theme is applied.